TestHelpers.Tests/MockDirectoryTests.cs

```diff
@@ -540,13 +540,9 @@ public void MockDirectory_CreMockDirectory_CreateDirectory_ShouldReturnDirectory
         }
 
```

Comment:
This is one example, I think there was at least one more.

```csharp
                 // These are no problems on Unix platforms
                 yield return @"..\";	       
                 yield return @"aaa\vv..\";	
                 yield return @"a..\b";	                 
```

I think I'd rather see this in another test and we could add this skip attribute

Reply:
6d58566 splits this to separate tests. I'm not sure (but also not very emotional) on what I like better 🤣

Reply:
haha I think the idea is better, anyway. You typically want to avoid any sort of logic within a test. 

For the approach, I don't think I'd have a test that calls another test with different input. I was more thinking just explicitly put the input in the test and run the test, even if the arrange/act are similar or exact.

So you'd have something like 

```csharp
[Skip on Linux]
public void MockDirectoryInfo_FullName_UNCPaths_ShouldReturnNormalizedPath()
{
  var input = object array of uncpaths {} // arrange
  // act
  // assert
}
```

Reply:
If that's still a little bit confusing, let me know and I could whip up an example. I just think its better to be a little be repetitive in tests vs. trying to avoid repeating yourself. When it comes to tests, I think it's generally better to show as much intent/be expressive as possible.

Reply:
Yeah, that's what I also didn't like. b6a744c tries to improve on this.

Reply:
Woo! I just saw one more, and then I think we're good. Thanks for this! 🎉